.github/workflows/ci-post-land.yml

```diff
@@ -0,0 +1,351 @@
+name: ci-post-land
+
+on:
+  create:
+    branches: [main, release-*]
+  push:
+    branches: [main, release-*] # "gha-test-*"
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  max_threads: 16
+  pre_command: cd /opt/git/diem/
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      changes-target-branch: ${{ steps.changes.outputs.changes-target-branch }}
+      changes-base-git-rev: ${{ steps.changes.outputs.changes-base-git-rev }}
+      changes-base-githash: ${{ steps.changes.outputs.changes-base-githash }}
+      changes-pull-request-number: ${{ steps.changes.outputs.changes-pull-request-number }}
+      rust-changes: ${{ steps.rust-changes.outputs.changes-found }}
+      base-image-changes: ${{ steps.base-image-changes.outputs.changes-found }}
+      update-osx-sccache: ${{ steps.environment.outputs.name }}
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          # This ensures that the tip of the PR is checked out instead of the merge between the base ref and the tip
+          # On `push` this value will be empty and will "do-the-right-thing"
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0 #get all the history!!!
+      - name: Git Hooks and Checks
+        run: ./scripts/git-checks.sh
+      - id: changes
+        name: determine changes
+        uses: diem/actions/changes@faadd16607b77dfa2231a8f366883e01717b3225
+        with:
+          workflow-file: ci-post-land.yml
+          github-token: ${{secrets.GITHUB_TOKEN}}
+      - id: rust-changes
+        name: find rust/cargo changes.
+        uses: diem/actions/matches@faadd16607b77dfa2231a8f366883e01717b3225
+        with:
+          pattern: '^documentation\|^docker\|^scripts\|^developers.diem.com'
+          invert: "true"
+      - id: base-image-changes
+        name: find dev-setup.sh/base docker image changes
+        uses: diem/actions/matches@faadd16607b77dfa2231a8f366883e01717b3225
+        with:
+          pattern: '.github/workflows/ci-post-land.yml\|.github/actions/dockerhub_login/action.yml\|docker/ci/github/Dockerfile\|scripts/dev_setup.sh\|rust-toolchain'
+
+  update-sccache-osx:
+    environment:
```

Comment:
Missing `needs: prepare`.  This job is not chained to the prepare job above.

Same for the rest of jobs below.

Reply:
ah good eyes, surprised the needs didn't result in an error

Reply:
Silent error in disguise :)  For the jobs that have an if-trigger on the prepare job's output, they were basically skipped.